src/server/services/hli/stadspas-types.ts

```diff
@@ -125,7 +130,67 @@ export interface StadspasTransactionQueryParams {
   budgetCode?: string;
 }
 
-export interface StadspasTransaction {
+// NOTE: The optional properties are not yet tested to be present in a response with transacties of `length > 0`.
```

Comment:
a little shorter: // NOTE: Optional properties unconfirmed for non-empty transaction lists.
// Currently sent raw to AmsApp; exact shape may not be critical yet.

Reply:
Done, dat leest wat beter. En ik heb de tweede regel een klein beetje aangepast